bricksrc/properties.py

```diff
@@ -5,27 +5,50 @@
 Defining properties
 """
 properties = {
+    # About expectedDomain and expectedRange: They replace temporarily
+    # RDFS.domin and RDFS.range respectively.  The .domain and .range
+    # properties confuse pre-inferencing of a SHACL validator without
+    # respecting disjointness.  Thus we want to remove them for now
+    # but still need the replacements to guide the shape generator.
```

Comment:
Its good to have this explanation here, but I would amend it slightly.

`rdfs:domain` and `rdfs:range` infer the type of the subject and object of a property, respectively. However, rather than imply the type of the subject or object by the use of the property, we want to verify that the property is used correctly given the types of the subject and object. Thus, we adopt the `brick:expectedDomain` and `brick:expectedRange` properties which imply nothing about the subject/object of the property they are annotating, but can be used to validate the use of Brick relationships.

Reply:
Please see how it looks now.